content/desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -166,6 +189,9 @@ The following `admin-settings.json` code and table provides an example of the re
 | `allowExperimentalFeatures`| | If `value` is set to `false`, experimental features are disabled.|
 | `allowBetaFeatures`| | If `value` is set to `false`, beta features are disabled.|
 | `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](../../../engine/reference/commandline/image_load.md) and receive an error if they try to.|
+| `filesharingAllowedDirectories` |  | Specify which paths your developers can add file shares to. Accepts `$HOME`, `$TMP`, or `$TEMP` as `path` variables. This specifices the allowed parent directory and subdirectories. If `sharedByDefault` is set to `true`, only the resolved paths should will be on the allow list. |
```

Comment:
Specify which paths your developers can add file shares to. Also accepts env $HOME, $TMP, or $TEMP as path variables. When a path is added, its subdirectories are allowed. If sharedByDefault is set to true, that path will be added upon factory reset or when DD first starts on the machine